### PR TITLE
[ENGDOCS-900] Organization audit log events

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Adding new organization audit log events.
https://deploy-preview-15325--docsdocker.netlify.app/docker-hub/audit-log/

Billing events updated in https://github.com/docker/docker.github.io/pull/15339

### Related issues (optional)
ENGDOCS-900

### CODE REVIEW

**Review by jerae-duffin — APPROVED** [review]
Thanks Craig! LGTM!

File: docker-hub/audit-log.md [hunk 1]
```diff
@@ -52,13 +52,20 @@ Refer to the following section for a list of events and their descriptions:
...
+| Team Created | Activities related to the creation of a team |
+| Team Updated | Activities related to the modification of a team |
 | Team Deleted | Activities related to the deletion of a team |
 | Team Member Added | Details of the member added to your team |
 | Team Member Removed | Details of the member removed from your team |
 | Team Member Invited | Details of the member invited to your team |
-| Organization Member Removed | Details about member removed from your organization |
-|  Organization Created| Activities related to the creation of a new organization |
+| Organization Member Removed | Details about the member removed from your organization |
+| Organization Created| Activities related to the creation of a new organization |
```
Comment (BuraChuhadar) [inline docker-hub/audit-log.md:62]: NIT: `Organization Created|` should be `Organization Created |`

File: docker-hub/audit-log.md [hunk 2]
```diff
@@ -52,13 +52,20 @@ Refer to the following section for a list of events and their descriptions:
...
+| Organization Settings Updated | Details related to the organization setting that was updated |
+| Registry Access Management enabled | Activities related to enabling Registry Access Management |
+| Registry Access Management disabled | Activities related to disabling Registry Access Management |
+| Registry Access Management registry added | Activities related to the addition of a registry |
+| Registry Access Management registry removed | Activities related to the removal of a registry |
+| Registry Access Management registry updated | Details related to the registry that was updated |
```
Comment (BuraChuhadar) [inline docker-hub/audit-log.md:69]: question: We are missing billing events, are you planning to add those? See billing events: https://www.notion.so/dockerinc/Audit-Log-List-8d1b669c158d49528f3931cf4f64c4f5

Comment (jerae-duffin) [reply docker-hub/audit-log.md:69]: @BuraChuhadar I have a PR created for Billing Events  https://github.com/docker/docker.github.io/pull/15339

**Review by ajthilakan — APPROVED** [review]

**Pull request merged**